src/main/java/org/terasology/projsndwv/commands/BeeCommands.java

```diff
@@ -146,17 +158,27 @@ public String birth(@Sender EntityRef client, @CommandParam(value = "count", req
         }
 
         EntityRef item = client.getComponent(ClientComponent.class).character.getComponent(CharacterHeldItemComponent.class).selectedItem;
-        if (item.getComponent(BeeComponent.class) == null || item.getComponent(MatedComponent.class) == null) {
-            return "Held item is not a mated bee.";
+        if (item.getComponent(BeeComponent.class) == null || item.getComponent(MatedComponent.class) == null || item.getComponent(BeeComponent.class).type != 2) {
+            return "Held item is not a queen.";
         }
 
         Iterator<GeneticsComponent> offspringGenetics = getGenome().combine(item.getComponent(GeneticsComponent.class), item.getComponent(MatedComponent.class).container.getComponent(GeneticsComponent.class));
 
-        for (int i = 0; i < count; i++) {
-            EntityRef drop = entityManager.create("Apiculture:bee");
+        EntityRef drop = entityManager.create("Apiculture:bee_princess");
+        drop.addComponent(offspringGenetics.next());
+        ItemComponent itemComponent = drop.getComponent(ItemComponent.class);
+        itemComponent.icon = TempBeeRegistry.getTextureRegionAssetForSpeciesAndType(drop.getComponent(GeneticsComponent.class).activeGenes.get(0), 1);
+        drop.addComponent(itemComponent);
+        drop.addComponent(TempBeeRegistry.getDisplayNameComponentForSpeciesAndType(drop.getComponent(GeneticsComponent.class).activeGenes.get(0), 1));
+        drop.send(new DropItemEvent(client.getComponent(ClientComponent.class).character.getComponent(LocationComponent.class).getWorldPosition()));
```

Comment:
Maybe see if you can place the items directly into the players inventory like the `give` command does.
Again, it's a temp command so if it's no worth it, that's fine

Reply:
This requires a dependency on Inventory. I believe I'll have to pull Inventory in later anyways, but that was the motivation for dropping in world as of present.